CHANGELOG.md

```diff
@@ -373,6 +378,7 @@ See the [Tendermint v0.34.7 SDK changelog](https://github.com/tendermint/tenderm
 
 [Unreleased]: https://github.com/cosmos/gaia/compare/v6.0.0...HEAD
```

Comment:
just confirm: the [unrelased] should always link to a relase, not a rc-release. so it should be `[Unreleased]: https://github.com/cosmos/gaia/compare/v6.0.0...HEAD` even though there is no `v6.0.0` yet, we only have `v6.0.0-rc1`